<launch>
  <arg name="image" default="/camera/rgb/image_rect_color" />
  <arg name="rects" default="/fcn_object_detector/rects" />
  
  <arg name="detection_threshold" default="0.5" />
  <arg name="min_boxes" default="3" />
  <arg name="nms_eps" default="0.2" />

  <arg name="pretrained_weights" default="$(find fcn_object_detector)/models/snapshot_iter_1000.caffemodel" />
  <arg name="deployment_prototxt" default="$(find fcn_object_detector)/models/deploy.prototxt" />
  
 <node pkg="fcn_object_detector" type="region_cnn_detector.py"
       name="region_cnn_detector" output="screen" >
   <remap from="image" to="$(arg image)" />
   <remap from="rects" to="$(arg rects)" />

   <param name="pretrained_weights" value="$(arg pretrained_weights)" />
   <param name="deployment_prototxt" value="$(arg deployment_prototxt)" />

   <param name="detection_threshold" value="$(arg detection_threshold)" />
   <param name="min_boxes" value="$(arg min_boxes)" />
   <param name="nms_eps" value="$(arg nms_eps)" />
 </node>
</launch>
